yarn.lock

```diff
@@ -1188,6 +1188,14 @@
     estree-walker "^1.0.1"
```

Comment:
All `yarn.lock` changes should be related to the changes in the package.json files in this PR.